cinema/serializers.py

```diff
@@ -1,6 +1,15 @@
+from django.db import transaction
```

Comment:
Code Style: Consider grouping imports using parentheses for better readability as per the checklist.